### DISCUSSION TITLE
The plugin doesn't support notes and sublists at the same level and doesn't support notes after sublists

### DISCUSSION THREAD

**Discussion by user30535** [vslinko/obsidian-outliner#344] [Ideas]
Outliner does not recognize the following list item:

```
- Apples are
  
  1. healthy
  2. delicious
  
  Therefore, I like apples!
```

When you press Ctrl + A in this list item, the expected result is that only this list item is selected:

![image](https://user-images.githubusercontent.com/108653772/178231831-34ab4257-afd5-458f-ad44-19ebf3cac5f1.png)

But the actual result is that the whole page is selected:

![image](https://user-images.githubusercontent.com/108653772/178232002-e76cb628-2dc1-42eb-bf94-3639f6179816.png)

"Stick the cursor to the content" also does not work with this example. Please notice the cursor:

![image](https://user-images.githubusercontent.com/108653772/178232458-a2f95664-6478-4f0e-aeea-ae2e51466152.png)

In addition, when I press Enter at the cursor here:

![image](https://user-images.githubusercontent.com/108653772/178232723-5e092599-b221-4e71-b72f-89e0552f231c.png)

The expected result is a new list item generated:

![image](https://user-images.githubusercontent.com/108653772/178233911-1e5e447d-a542-4ff4-95a1-79bf366c4c1e.png)

but the actual result is a new line generated:

![image](https://user-images.githubusercontent.com/108653772/178232906-40ca8802-2e48-4dda-8ff2-f7a3035ceadb.png)

In the above example, spaces are used for indentation of the nested list. The same problems happen if a tab is used for indentation:

```
- Apples are
  
	1. healthy
	2. delicious
  
  Therefore, I like apples!
```

Outliner does not recognize the list no matter spaces or tabs are used for indentation.

**Desktop (please complete the following information):**
 - OS: Win 10
 - Obsidian Version: 0.14.15

**Comment by vslinko** [comment]
@user30535 thank you for the feedback!

Yes, currently the plugin doesn't support notes and sublists at the same level and doesn't support notes after sublists.
You can only create lists like this:

```
- Apples are
  Therefore, I like apples!
  1. healthy
  2. delicious
```

I'll keep your feedback as a feature request.

**Reply by user30535** [reply]
Thank you so much. This is a much needed feature for me :)

**Comment by user30535** [comment]
Could anyone please help add this feature? It is very important for me.
